samples/function_chaining_custom_status/.funcignore

```diff
@@ -0,0 +1,5 @@
+.git*
+.vscode
+local.settings.json
+test
```

Comment:
I don't think we're supposed to push this, right? 😸 
Since this is manged by the VSCode extension

Reply:
Woah. thanks for noticing. will remove it.

Reply:
.vscode is created by VS code and I will keep it in .funcignore. I have removed py3_env